app/build.gradle.kts

```diff
@@ -39,6 +39,8 @@ firebaseAppDistribution {
 dependencies {
     // Most of the dependencies are coming from the convention plugin to avoid duplication with `:automotive` module.
     "fullImplementation"(libs.car.projected)
+
+    implementation(libs.androidx.lifecycle.service)
```

Comment:
I think this should go in `AndroidApplicationDependenciesConventionPlugin` otherwise automotive module won't have it.